restore/restore.go

```diff
@@ -440,6 +444,13 @@ func editStatementsRedirectSchema(statements []toc.StatementWithType, redirectSc
 			replaced = true
 		}
 
+		// Statistic statements needs one schema replacements. We replace the last occurence to avoid a very small
+		// chance that schema name was in the statistic data itself, that we do not want to alter.
+		if strings.HasPrefix(statement, "\n\nDELETE FROM pg_statistic") || strings.HasPrefix(statement, "\n\nINSERT INTO pg_statistic") {
```

Comment:
Could you add test for it, because somebody may change \n\n to something (\n or \n\n\n) in generate statement step?

Reply:
may be use condition like `ObjectType == OBJ_STATISTICS`?